packages/edit-post/src/store/selectors.js

```diff
@@ -223,7 +223,25 @@ export function getMetaBoxes( state ) {
  * @return {string[]} Active meta box locations.
  */
 export function getActiveMetaBoxLocations( state ) {
-	return state.activeMetaBoxLocations;
+	return Object.keys( state.metaBoxes.locations )
```

Comment:
Should we use memoization here?

Reply:
Yes, this will cause `OptionsModal` to update unnecessarily.